src/components/video-editor/timeline/ItemGlass.module.css

```
@@ -1,194 +1,187 @@
-  margin: 1px 0;
-  backdrop-filter: blur(4px);
-  -webkit-backdrop-filter: blur(4px);
-  transition: all 0.2s cubic-bezier(0.4, 0, 0.2, 1);
+	position: relative;
+	border-radius: 8px;
+	background: rgba(37, 99, 235, 0.38);
+	border: 1px solid rgba(37, 99, 235, 0.5);
+
+	transition:
+		background 0.15s ease,
+		border-color 0.15s ease;
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)